<launch>

    <node pkg="green_fundamentals" name="odometry_node" type="odometry"/> 

    <node pkg="green_fundamentals" name="cmd_vel_node" type="cmd_vel"/>

    <node pkg="green_fundamentals" name="strict_controller_node" type="strict_controller"/>
  
  </launch>
  
